examples/clients/abe/examplepb_a_bit_of_everything.go

```diff
@@ -14,6 +14,7 @@ import (
 	"time"
 )
 
+// Intentionaly complicated message type to cover much features of Protobuf.
```

Comment:
Nit: s/much/many/

Reply:
Yeh I intentionally didn't change this existing typo but may as well :D